apps/_pomodoro_app/realtime/application/AbstractAuthMiddleware.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * A class to handle the auth of the socket
+ */
+class AbstractAuthMiddleware {
+
+    /**
+     * This buddy is a middleware, that handles auth in socket.io
+     * More here: https://socket.io/docs/migrating-from-0-9/
+     * @param socket
+     * @param next {function}
+     */
+    static auth(socket, next) {}
```

Comment:
why it's static?

Reply:
why it's not a function If it's really static

Reply:
I couldn't imagine a case when we need a different auth middlewares for different requests. This was made a class only to simplify the testing process and enhance the

Reply:
It was not made a function because we might want to make auth by `POST` request to keystone

Reply:
Anyway this buddy was rewritten